### DISCUSSION TITLE
Crud -> Create New Record (Modal Popup)

### DISCUSSION THREAD

**Discussion by NeguinhoAdmin** [Laravel-Backpack/community-forum#1121] [General] [answered]
Hi,

"upon click on create new record button" I wonder, how could I have a Modal showing some information having checkboxes (non data driven) kinda affirmation before making a record. 

The information showing the prerequisites as a reminder before proceed new record.

Can someone tell me is there native way to do that?

Thank you

**Comment by jcastroa87** [comment, marked as answer]
Hello @NeguinhoAdmin 

There is no native feature for this, last time someone ask something similar, the solution is play with [Javascript API](https://backpackforlaravel.com/docs/6.x/crud-fields-javascript-api) to hide and show fields.

Cheers.

**Comment by jcastroa87** [comment]
Due to not activity, I will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
